scripts/upgrade_admin_server.sh

```diff
@@ -78,6 +78,9 @@ upgrade_admin_server()
 
     trap cleanup INT EXIT
 
+    # lock crowbar-ui package until upgrade is finished
```

Comment:
could you please expand the comment? Something like _... so that the package intended for next product upgrade is not installed_ ?